config.sample.yaml

```diff
@@ -8,6 +8,10 @@ management:
   mgmt_storage_account_name: __CHANGE_ME__
   terraform_state_container_name: tfstate
   acr_name: __CHANGE_ME__
+  # ID of external Key Vault to store CMKs in (only required if enable_cmk_encryption is true)
```

Comment:
I think worth adding to documentation as well : https://microsoft.github.io/AzureTRE/latest/tre-admins/environment-variables/

Reply:
thanks! added